frontend/src/components/slider/index.jsx

```diff
@@ -0,0 +1,121 @@
+import { createSignal, createEffect, on } from "solid-js";
+import { styled } from "solid-styled-components";
+import arrow from "../../assets/arrow.svg";
+
+const Slider = (props) => {
+  // useState
+  const [imageIndex, setImageIndex] = createSignal(0);
+
+  // refs
+  let buttonNext, buttonPrev;
+
+  // It will be in props in feature
+  const imagesList = () => props.imagesList || [];
```

Comment:
The `imageList` should be a valid list in any case. It should not be `undefined` or `null`